### ISSUE TITLE
es6 promise dependency throwing error with cryptic message

### ISSUE THREAD

**Issue by bsouthga** [CrossLead/netsuite-js#3]
```
Array
    node_modules/bugsnag/lib/error.js:69:32 Error
    node_modules/bugsnag/lib/error.js:80:20 buildErrors
    node_modules/bugsnag/lib/bugsnag.js:99:34 notify
    app/server/_shared/config/agenda.js:20:209 none
    native generator.js:38:30 throw
    app/server/_shared/config/agenda.js:1:3310 step
    app/server/_shared/config/agenda.js:1:3494 none
    node_modules/netsuite-js/node_modules/es6-promise/dist/es6-promise.js:331:16 lib$es6$promise$$internal$$tryCatch
    node_modules/netsuite-js/node_modules/es6-promise/dist/es6-promise.js:343:17 lib$es6$promise$$internal$$invokeCallback
    node_modules/netsuite-js/node_modules/es6-promise/dist/es6-promise.js:314:11 lib$es6$promise$$internal$$publish
    node_modules/netsuite-js/node_modules/es6-promise/dist/es6-promise.js:264:7 lib$es6$promise$$internal$$publishRejection
    node_modules/netsuite-js/node_modules/es6-promise/dist/es6-promise.js:125:9 lib$es6$promise$asap$$flush
    node.js:419:9 nextTickCallbackWith0Args
    node.js:389:13 _tickDomainCallback
```

**Comment by yangchristian** [comment]
Will remove the `es6-promise` polyfill for now in hopes of unmasking the underlying error.

**Comment by yangchristian** [comment]
Resolved with 59f2291363aeeea2ff73e79d933c2e1b67d00cdb
